extensions/wikia/AdEngine/AdEngine.i18n.php

```diff
@@ -1,893 +1,585 @@
-
-/** Tatar (Cyrillic script) (татарча)
- */
-$messages['tt-cyrl'] = array(
-	'adengine-advertisement' => 'реклама',
-);
-
-/** Ukrainian (українська)
- * @author Andriykopanytsia
- * @author Erami
- * @author Mykola Swarnyk
- * @author Ua2004
- * @author Капитан Джон Шепард
- */
-$messages['uk'] = array(
-	'adengine-advertisement' => 'реклама',
-	'adengine-exitstitial-title-template' => 'Ти йдеш з $1...',
-	'adengine-exitstitial-redirecting' => 'Ваш браузер переадресує вас на запитувану вами сторінку за декілька секунд.',
-	'adengine-exitstitial-go-back' => 'Бажаєте повернутися?',
-	'adengine-exitstitial-button' => 'Пропустити це оголошення',
-	'adengine-desc' => 'Вікі оголошення',
-);
-
-/** Vietnamese (Tiếng Việt)
- * @author Baonguyen21022003
- * @author Dinhxuanduyet
- * @author KhangND
- * @author Max20091
- * @author Xiao Qiao
- */
-$messages['vi'] = array(
-	'adengine-advertisement' => 'quảng cáo',
-	'adengine-exitstitial-title-template' => 'Bạn đang rời khỏi $1…',
-	'adengine-exitstitial-redirecting' => 'Trình duyệt của bạn sẽ chuyển hướng bạn đến trang web mà bạn yêu cầu trong vài giây nữa.',
-	'adengine-exitstitial-go-back' => 'Bạn muốn quay trở lại?',
-	'adengine-exitstitial-button' => 'Bỏ qua quảng cáo này',
-	'adengine-desc' => 'Cơ chế quảng cáo của Wikia',
-);
-
-/** Waray (Winaray)
- * @author Harvzsf
- */
-$messages['war'] = array(
-	'adengine-exitstitial-title-template' => 'Naíwas ka na han $1...',
-	'adengine-exitstitial-go-back' => '¿Karúyag mo bumálik?',
-);
-
-/** Mingrelian (მარგალური)
- * @author Silovan
- */
-$messages['xmf'] = array(
-	'adengine-exitstitial-button' => 'გეგშიტეთ თე რეკლამა',
-);
-
-/** Chinese (中文)
- * @author Shangkuanlc
- */
-$messages['zh'] = array(
-	'adengine-exitstitial-button' => '跳過廣告',
-);
-
-/** Simplified Chinese (中文（简体）‎)
- * @author Byfserag
- * @author Duolaimi
- * @author Liuxinyu970226
- * @author Yfdyh000
- * @author 御坂美琴
- */
-$messages['zh-hans'] = array(
-	'adengine-advertisement' => '广告',
-	'adengine-exitstitial-title-template' => '你正在离开$1...',
-	'adengine-exitstitial-redirecting' => '您的浏览器将在几秒钟后重定向至您请求的页面。',
-	'adengine-exitstitial-go-back' => '想要返回吗？',
-	'adengine-exitstitial-button' => '跳过这个广告',
-	'adengine-desc' => 'Wikia的广告引擎',
-);
-
-/** Traditional Chinese (中文（繁體）‎)
- * @author Citizen01
- */
-$messages['zh-hant'] = array(
-	'adengine-advertisement' => '廣告',
-	'adengine-exitstitial-title-template' => '您正離開 $1 ……',
-	'adengine-exitstitial-redirecting' => '您的瀏覽器幾秒鐘內將重新導向到您請求的頁面。',
-	'adengine-exitstitial-go-back' => '想要返回嗎？',
-	'adengine-exitstitial-button' => '跳過廣告',
-	'adengine-desc' => 'Wikia 的廣告引擎',
-);
+$messages = [ ];
+
+$messages["en"] = [
+	"adengine-advertisement" => "advertisement",
+	"adengine-exitstitial-title-template" => "You're leaving $1…",
+	"adengine-exitstitial-redirecting" => "Your browser will redirect you to the page you requested in a few seconds.",
+	"adengine-exitstitial-go-back" => "Want to go back?",
+	"adengine-exitstitial-button" => "Skip this Ad",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "adengine-exitstitial-button" 69 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)